Source/URLEncodedFormEncoder.swift

```diff
@@ -294,11 +294,18 @@ public final class URLEncodedFormEncoder {
 
         var localizedDescription: String {
             switch self {
-            case let .invalidRootObject(object): return "URLEncodedFormEncoder requires keyed root object. Received \(object) instead."
+            case let .invalidRootObject(object):
+                return "URLEncodedFormEncoder requires keyed root object. Received \(object) instead."
             }
         }
     }
 
+    /// Whether or not to sort the encoded key value pairs.
+    ///
+    /// - Note: This setting ensures a consistent ordering for all encodings of the same parameters. When set to `false`,
+    ///         encoded `Dictionary` values may have a different encoded order each time they're encoded due to
+    ///       ` Dictionary`'s random storage order.
```

Comment:
I think this `Note` is a bit outdated since you've switched over to a tuple array. Isn't it now going to be either the order the parameters were provided or alphabetically sorted?

Reply:
No, this note refers to `Dictionary`s being encoded, which will still have a variable order between app launches. Let me know if there’s a better way to say that here.